examples/httpserver_redirects.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2022 Dan Halbert for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
+import socketpool
+import wifi
+
+from adafruit_httpserver import Server, Request, Response, Redirect, NOT_FOUND_404
+
+
+pool = socketpool.SocketPool(wifi.radio)
+server = Server(pool, debug=True)
+
+REDIRECTS = {
+    "google": "https://www.google.com",
+    "adafruit": "https://www.adafruit.com",
+    "circuitpython": "https://circuitpython.org",
+}
+
+
+@server.route("/blinka")
+def redirect_blinka(request: Request):
+    """
+    Always redirect to a Blinka page as permanent redirect.
+    """
+    return Redirect(request, "https://circuitpython.org/blinka", permanent=True)
+
+
+@server.route("/<slug>")
+def redirect_other(request: Request, slug: str = None):
+    """
+    Redirect to a URL based on the slug.
+    """
+
+    if slug is None or not slug in REDIRECTS:
+        return Response(request, text="Unknown redirect", status=NOT_FOUND_404)
```

Comment:
This Response is raising an exception when it attempts to init and therefore doesn't make it back to the browser correctly. 

The exception raised is:
```
Traceback (most recent call last):
  File "/lib/adafruit_httpserver/server.py", line 327, in poll
  File "/lib/adafruit_httpserver/server.py", line 266, in _handle_request
  File "/lib/adafruit_httpserver/route.py", line 154, in wrapped_handler
  File "code.py", line 37, in redirect_other
TypeError: unexpected keyword argument 'text'
```

Reply:
Fixed, I must have missed it during refactoring. Nice catch.